aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -589,6 +606,30 @@ TODO bring back with generic rotation capability
         create_authorized_signer(&charlie, signer::address_of(&alice));
     }
 
+    #[test(bob = @0x345)]
+    public entry fun test_valid_revoke_signer_capability(bob: signer) acquires Account {
+        let pk = x"f66bf0ce5ceb582b93d6780820c2025b9967aedaa259bdbb9f3d0297eced0e18";
+        let alice = create_account_from_ed25519_public_key(pk);
+        create_account(signer::address_of(&bob));
+
+        let valid_signature = x"cd181d65eb31193dcf1627fc0cc04208f66e7f243facc840830eaa458b176de570f73b661c127d98bc276c5a07ab242734b4d656163a86803561c0b9d9d01d0c";
```

Comment:
Can you add instructions here on how others can reproduce this signature? What did you use?

Reply:
I used Rust to mock a transaction - will add a note!